### PR TITLE
Document _rw_kafka_timestamp

### LINKED ISSUE #519: Document `_rw_kafka_timestamp`
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/7275
https://github.com/risingwavelabs/risingwave/pull/7150

### Which part(s) of the docs might be affected or should be updated? And how?

'Ingest data from Kafka' -> Add descriptions of this field and mention that users can use this field to set the scope of messages they want to ingest. 

### Reference

_No response_

### PR DESCRIPTION
**Pull request by emile-00**
## Info
- **Description**: 
Document _rw_kafka_timestamp by adding new `is_timestamp` field under Ingest data from Kafka page.

- **Preview**: 
[ Paste the preview link to the edited page here. ]

- **Related code PR**: 
https://github.com/risingwavelabs/risingwave/pull/7275
https://github.com/risingwavelabs/risingwave/pull/7150

- **Related doc issue**: 
https://github.com/risingwavelabs/risingwave-docs/issues/519

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/create-source/create-source-kafka.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ For materialized sources with primary key constraints, if a new data record with
 |properties.group.id	|Required. Name of the Kafka consumer group	|
 |scan.startup.mode|Optional. The offset mode that RisingWave will use to consume data. The two supported modes are `earliest` (earliest offset) and `latest` (latest offset). If not specified, the default value `earliest` will be used.|
 |scan.startup.timestamp_millis|Optional. RisingWave will start to consume data from the specified UNIX timestamp (milliseconds). If this field is specified, the value for `scan.startup.mode` will be ignored.|
+|is_timestamp | Optional. If true, RisingWave will create the `_rw_kafka_timestamp` column to store the Kafka timestamp of each message. To query messages based on this timestamp, use a conditional statement like `_rw_kafka_timestamp > '2023-01-01 00:00:00.00'.|
```
Comment (liurenjie1024) [inline docs/create-source/create-source-kafka.md]: I'm not aware that we add support for this `is_timestamp` option. The `_rw_kafka_timestamp` is a virtual column extending source schema, e.g. it always exists. That's, if user wants to store kafka message's timestamp, they just include it in ddl statement:
```
create materialized view v1 as 
select _rw_kafka_timestamp, col1 from source1;
```

When querying directly on kafka, user can use `_rw_kafka_timestamp` column to skip scanning data. For example if user only wants to scan latest 10 minutes data in kafka:
```
select * from source1 where _rw_kafka_timestamp > now() - interval '10 minute'
```

**Review by CharlieSYH — APPROVED** [review]

File: docs/create-source/create-source-kafka.md [hunk 2]
```diff
@@ -123,6 +123,26 @@ ROW SCHEMA LOCATION 'https://demo_bucket_name.s3-us-west-2.amazonaws.com/demo.pr
 </TabItem>
 </Tabs>
 
+## Query Kafka timestamp
+
+For each Kafka source created, the virtual column, `_rw_kafka_timestamp`, will also exist. This column includes the timestamp of the Kafka message.
+
+To store this column, users can use the following query.
```
Comment (hengm3467) [inline docs/create-source/create-source-kafka.md]: Store the column might not be an accurate description. I suggest rewording this sentence to 'You can include this column in your views or materialized views to display timestamps. Here is an example:'

File: docs/create-source/create-source-kafka.md [hunk 3]
```diff
@@ -123,6 +123,26 @@ ROW SCHEMA LOCATION 'https://demo_bucket_name.s3-us-west-2.amazonaws.com/demo.pr
...
+
+```sql
+CREATE MATERIALIZED VIEW v1 AS
+SELECT _rw_kafka_timestamp, col1
+FROM source_name;
+```
+
+If directly querying from the source, users can use `_rw_kafka_timestamp` to query messages sent within a specific time period. For example, the following query only selects messages sent in the past 10 minutes.
```
Comment (hengm3467) [inline docs/create-source/create-source-kafka.md]: General comment: Please replace 'users' with 'you' or 'You'.

Comment (hengm3467) [reply docs/create-source/create-source-kafka.md]: ... to query messages - > ... to filter messages

File: docs/create-source/create-source-kafka.md (hunk 1 above)
Comment (emile-00) [reply docs/create-source/create-source-kafka.md]: @liurenjie1024 I've updated the description according to your suggestions. Please take a look again, thank you!

**Review by liurenjie1024 — APPROVED** [review]
LGTM
